service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,360 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!-- Order & Related Services -->
+    <service verb="create" noun="CreateOrder" displayName="Create the order">
+        <in-parameters>
+            <parameter name="orderName" required="true"/>
+            <parameter name="currencyUomId" required="true"/>
+            <parameter name="salesChannelEnumId" required="true"/>
+            <parameter name="statusId" required="true"/>
+            <parameter name="productStoreId" required="true"/>
+            <parameter name="placedDate" required="true"/>
+            <parameter name="approvedDate" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Party" list="orderCreateList">
+                <econdition field-name="partyId" value="CustJqp"/>
+            </entity-find>
+            <if condition="orderCreateList">
+                <then>
+                    <service-call name="create#mantle.order.OrderHeader" in-map="context" out-map="context"/>
+                </then>
+            </if>
+        </actions>
+    </service>
+
+
+    <service verb="create" noun="CreateOrderPart" displayName="Create the order Part">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="partName" required="true"/>
+            <parameter name="facilityId" required="true"/>
+            <parameter name="shipmentMethodEnumId" required="true"/>
+            <parameter name="customerPartyId" required="true"/>
+            <parameter name="item_details" type="List">
+                <parameter name="orderEntry" type="Map">
+                    <parameter name="productId"/>
+                    <parameter name="itemDescription"/>
+                    <parameter name="quantity"/>
+                    <parameter name="unitAmount"/>
+                </parameter>
+            </parameter>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="orderPartSeqId" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderCreateList">
```

Comment:
We can also use entity-find-count here to check if a record is existing. It is less expensive than retrieving the entire record.